lib/web_ui/dev/roll_fallback_fonts.dart

```diff
@@ -346,13 +344,14 @@ OTHER DEALINGS IN THE FONT SOFTWARE.
                 'Chrome/112.0.0.0 Safari/537.36'
       });
       final String cssString = response.body;
-      final StyleSheet stylesheet = csslib.parse(cssString);
-      final UriCollector uriCollector = UriCollector();
-      stylesheet.visit(uriCollector);
+      // Match the patterns that look like:
+      // `src: url(...some url...)`
+      final r = RegExp(r'src:\s*url\(([^)]+)\)');
       int familyCount = 0;
       // Give each font shard a unique family name.
-      for (final Uri uri in uriCollector.uris) {
+      for (final match in r.allMatches(cssString)) {
```

Comment:
Do we need to check that the `match` is a proper font URI? (like ending in known extensions or something?)

Reply:
Instead of doing that in the `if`, I'll make the regex more restrictive (i.e. starts with `https?://` and ends with `\.woff2`).

Reply:
SGTM! Something I like to do when I write a complex RegEx is to create it here:

* https://regexr.com

And save it (they support "Github login") with some Tests there. It's pretty fancy for docs!